src/test/kotlin/com/forgerock/securebanking/support/Users.kt

```diff
@@ -1,43 +1,57 @@
 package com.forgerock.securebanking.support
 
 import com.forgerock.securebanking.framework.configuration.DOMAIN
+import com.forgerock.securebanking.framework.constants.IAM
 import com.forgerock.securebanking.framework.http.fuel.responseObject
+import com.forgerock.securebanking.framework.utils.adminAuthentication
+import com.forgerock.securebanking.framework.utils.getIDMAdminAuthCode
+import com.forgerock.securebanking.framework.utils.getIDMAdminToken
 import com.forgerock.securebanking.tests.functional.directory.UserRegistrationRequest
 import com.github.kittinunf.fuel.Fuel
 import com.github.kittinunf.fuel.core.isSuccessful
 import com.github.kittinunf.fuel.gson.jsonBody
+import com.google.gson.Gson
 import org.apache.http.client.utils.URIBuilder
 import java.net.URLEncoder
 import java.util.*
 
-
 private fun registerUser(realm: String): UserRegistrationRequest {
-    val user = UserRegistrationRequest("fortest_" + UUID.randomUUID(), "password")
-    val (_, response, result) = Fuel.post("https://am.$DOMAIN/json/realms/root/realms/$realm/selfservice/userRegistration?_action=submitRequirements")
-        .header("Accept-API-Version", "protocol=1.0,resource=1.0")
-        .jsonBody(user)
+    val cookie = adminAuthentication()
+    val code = getIDMAdminAuthCode(cookie)
+    val accessToken = getIDMAdminToken(cookie, code)
+
+    val username = "fortest_" + UUID.randomUUID()
+    val user = UserRegistrationRequest(username, "Password@1", username, username, "mail@forgerock.com")
+    val (_, response, result) = Fuel.post("$IAM/openidm/managed/user?_action=create")
```

Comment:
Could we move the password value to a property or environment variable to change it more easily? just in case that we will need update that value depending of the environment.